packages/core-database/lib/wallet-manager.js

```diff
@@ -376,6 +376,41 @@ module.exports = class WalletManager {
     return data
   }
 
+  /**
+   * Determine the list of active delegates for the given round.
+   * @param  {Number} maxDelegates
+   * @return {Array}
+   */
+  determineActiveDelegates (maxDelegates) {
+    const mapWallets = wallets => wallets.map(wallet => ({
+      publicKey: wallet.vote || wallet.publicKey,
+      balance: wallet.balance ? wallet.balance.toFixed() : 0
```

Comment:
`toFixed()` will return String. So, to be consistent, maybe replace `0` with `"0"` or use `Number(wallet.balance.toFixed())`